entity-framework/core/providers/cosmos/index.md

```diff
@@ -7,6 +7,9 @@ uid: core/providers/cosmos/index
 ---
 # EF Core Azure Cosmos DB Provider
 
+> [!WARNING]
+> Extensive work has gone into the Cosmos DB provider in 9.0. In order to improve the provider, a number of high-impact breaking changes had to be made; if you are upgrading an existing application, please read the [breaking changes section](xref:core/what-is-new/ef-core-9.0/breaking-changes#cosmos-breaking-changes) carefully.
+
 This database provider allows Entity Framework Core to be used with Azure Cosmos DB. The provider is maintained as part of the [Entity Framework Core Project](https://github.com/dotnet/efcore).
 
 It is strongly recommended to familiarize yourself with the [Azure Cosmos DB documentation](/azure/cosmos-db/introduction) before reading this section.
```

Comment:
Add a note below, for `EnsureCreatedAsync`:

Azure Cosmos DB SDK does not support RBAC for management plane operations in Azure Cosmos DB. Use [Azure Management API](https://learn.microsoft.com/en-us/azure/cosmos-db/role-based-access-control#check-list-before-enabling) instead of `EnsureCreatedAsync` with RBAC.